json-json4s/shared/src/main/scala/rapture/json-json4s/ast.scala

```diff
@@ -78,7 +77,10 @@ private[json4s] object Json4sAst extends JsonBufferAst {
 
   def setObjectValue(obj: Any, name: String, value: Any): Any = {
     val contents = (name, value) :: obj.asInstanceOf[JObject].obj.filter(_._1 != name)
-    JObject(contents map { case (k: String, v: JValue) => k -> v })
+    JObject(contents map {
+      case (k: String, v: JValue) => k -> v
+      case _ => ???
```

Comment:
This is just changing from a match error to a not-implemented error, right? Is there some benefit to that?